changes/909.bugfix.rst

```diff
@@ -0,0 +1 @@
+``briefcase create`` no longer fails if a template doesn't provide an empty ``app`` folder.
```

Comment:
I'm curious...this is a second recent fix to an unreleased bug, wanted to ask "officially"....should unreleased bugs be a `bugfix` change note or `misc`?

Reply:
That's a good point; it only needs to be listed as a bugfix if the observed problem is in a public release. Otherwise the context for the release note won't make any sense.